src/typical.jl

```diff
@@ -103,7 +103,17 @@ function _trmequal(t1::FunctionTerm, t2::FunctionTerm)
            t1.forig == t2.forig
 end
 
-function typicalterm(term::AbstractTerm, context::MatrixTerm, model_matrix; typical=mean)
+# XXX should probably drop the kwarg version of this
+# and skip the redirect, but that's Technically Breaking™
+# likewise add an AbstractMatrix type restriction to model_matrx
+function typicalterm(term::AbstractTerm, context::MatrixTerm,
+                     model_matrix; typical=mean)
+    return typicalterm(term, context, model_matrix, typical)
+end
```

Comment:
use `@deprecate` then?

```suggestion
@deprecate typicalterm(term::AbstractTerm, context::MatrixTerm, model_matrix; typical=mean) typicalterm(term, context, model_matrix, typical)
```

dunno if this actually works as written but that's the idea...